integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -447,25 +407,6 @@ void testComplexType() {
     checkTableReadWrite(tableInfo);
   }
 
-  @Test
-  void testCreateDatasourceFormatPartitionTable() {
```

Comment:
I perfer to keep this test in common because it could be shared by Hive and Iceberg.  Iceberg doesn't support partition in this pr, we could add `supportsPartition` to enable this like `supportsSparkSQLClusteredBy`

Reply:
updated.